devops/Vagrantfile

```diff
@@ -0,0 +1,21 @@
+Vagrant.require_version ">= 1.7.0"
+
+Vagrant.configure(2) do |config|
+
+  config.vm.provider "virtualbox" do |v|
+    v.name = "multinet-ubuntu-arango-test"
+    v.memory = 4096
+    v.cpus = 2
+  end
+
+  config.vm.box = "ubuntu/bionic64"
+  config.ssh.insert_key = false
+
+  config.vm.provision "ansible" do |ansible|
+    ansible.verbose = "v"
```

Comment:
We should probably omit this line (and let the user decide on their own level of verbosity).

```suggestion
```